examples/hybrid/radiation_utilities.jl

```diff
@@ -119,23 +119,19 @@ function rrtmgp_model_cache(
         center_pressure = NaN, # initialized in callback
         center_temperature = NaN, # initialized in callback
         surface_temperature = NaN, # initialized in callback
-        surface_emissivity = FT(1),
-        direct_sw_surface_albedo = FT(0.38),
-        diffuse_sw_surface_albedo = FT(0.38),
+        surface_emissivity = 1,
```

Comment:
How are these defined as floats here, but later as fields?

also, not seeing these defaults being used - later on, the values of 1 and 0.38 are used, but hardcoded.

Reply:
Yes we should remove these properties from the cache in the staggered_nonhydrostatic_model. Those are not being used. Here is where these properties are defined (hardcoded). I think we chose not to use parameter set for now, @dennisYatunin what is the reason?

Reply:
And I guess in the coupled runs rrtmgp knows to look for albedo and emissivity in the cache?

Reply:
Yes.

Reply:
Since ClimaParameters is going to change significantly very soon, we should probably avoid adding things there unless absolutely necessary. I'm not sure if `surface_emissivity = 1` is a parameter that is urgently needed across CliMA, so it's probably fine to leave this hardcoded for now.

The reason these values are constants here is that I wrote the `RRTMGPModel` interface to be as user-friendly as possible. If the user specifies a constant, then the entire field is set to that constant. If the user specifies a 1D array for a 3D field, that array is used to set each column. The full set of possible inputs is described in the `RRTMGPModel` docstring.

In order to modify, the surface emissivity of each band from the coupler, you will probably want to do something like
```
number_of_bands = size(rrtmgp_model.surface_emissivity, 1)
for band in 1:number_of_bands
    surface_emissivity = RRTMGPI.array2field(rrtmgp_model.surface_emissivity[band, :], bottom_level_space)
    @. surface_emissivity = <broadcast expression for surface emissivity for current wavelength band>
end
```